### ISSUE TITLE
[CoE Starter Kit - Feature]: Add customized emails to the admin view app

### ISSUE THREAD

**Issue by ashlega** [microsoft/coe-starter-kit#4461]
### Is your feature request related to a problem? Please describe.

Can we have "customized emails" table added to the admin view app? Might be helpful to also have it added to the sitemap (possibly to the "Admin" area?).

Right now I can't get to the customized emails directly from the admin view app (for example, that table does not show up in the the updated advanced find search since it's not part of the app). Have to go to the "classic" advanced find to look it up.

### Describe the solution you'd like

- Add customized emails table to the admin view app
- Add it to the sitemap navigation (possibly to the admin area)

**Comment by Jenefer-Monroe** [comment]
Can you explain what properties of the table you would want to see / manipulate there? 
We expose this in the Admin - Command Center today.

**Comment by ashlega** [comment]
Ok, I did not realize it's available in the Admin Command Center, but I can see it there now, thank you. Off the top of my head, it seems it would still be nice to just be able to stay in the MDA instead of switching to another app.... but it's just a matter of personal preference then (well, that said, command center does not replace Admin View MDA, since I'd still need to be in the MDA to see compliance responses, for example). 

PS. To answer your question - the reason I had to access that table is that I needed to update compliance emails (email body)
